### PR TITLE
Feature add util tools

### PR DESCRIPTION
**Pull request by shc261392**
## Chore
- Added icons used in Capture Eye and deploy scripts for staging code and icons
- Added `knip` for checking unused export

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +67,7 @@
     "rollup": "^2.73.0",
     "rollup-plugin-summary": "^1.4.3",
     "rollup-plugin-terser": "^7.0.2",
+    "ts-prune": "^0.10.3",
```
Comment (olgahaha) [inline package.json:70]: just a heads-up that ts-prune is going into [maintenance mode](https://github.com/nadeesha/ts-prune?tab=readme-ov-file#-ts-prune-is-going-into-maintanence-mode). We might want to explore alternatives in the future.

File: src/constant.ts [hunk 2]
```diff
@@ -32,11 +29,13 @@ interface ConstantType {
   layout: Layout;
 }
 
-export interface Layout {
+interface Layout {
   original: string;
   curated: string;
 }
 
+const NumbersCdnUrl = 'https://static-cdn.numbersprotocol.io';
```
Comment (olgahaha) [inline src/constant.ts:37]: how about starting with lowercase like the other consts?

File: package.json (hunk 1 above)
Comment (shc261392) [reply package.json]: Change to use `knip` recommended in ts-prune repo

**Comment by olgahaha** [comment]
could you please correct the GitHub description?

**Review by olgahaha — APPROVED** [review]

**Pull request merged**